packages/clarity-js/src/layout/schema.ts

```diff
@@ -0,0 +1,56 @@
+import { Dimension, Metric, Setting } from "@clarity-types/data";
+import { Constant, JsonLD } from "@clarity-types/layout";
+import * as dimension from "@src/data/dimension";
+import * as metric from "@src/data/metric";
+
+/* JSON+LD (Linked Data) Recursive Parser */
+export function ld(json: any): void {
+    for (let key of Object.keys(json)) {
+        let value = json[key];
+        if (key === JsonLD.Type && typeof value === "string") {
+            value = value.toLowerCase();
+            /* Normalizations */
+            value = value.indexOf(JsonLD.Article) >= 0 || value.indexOf(JsonLD.Posting) >= 0 ? JsonLD.Article : value;
```

Comment:
Recommend values to normalize to article are:

* NewsArticle
* BlobPosting

See [https://schema.org/docs/full.html](https://schema.org/docs/full.html)

If it's just the constant names that differ can you please update them to the full name so it's easier to match?